sky/templates/kubernetes-sshjump.j2

```diff
@@ -0,0 +1,74 @@
+{

```

Comment:
Can we have this file as a YAML instead of a json? Trying to have something that can also be easily used with `kubectl apply -f ` for debugging/testing (of course, after replacing template variables).